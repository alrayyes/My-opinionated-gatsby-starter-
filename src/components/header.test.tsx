import renderer from "react-test-renderer"
import Header from "./header"
import React from "react"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
