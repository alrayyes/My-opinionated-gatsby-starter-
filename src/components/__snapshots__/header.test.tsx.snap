// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-1pcnel4-Header{background:rebeccapurple;margin-bottom:1.45rem;}",
      }
    }
    data-emotion-css="1pcnel4-Header"
  />,
  <header
    className="css-1pcnel4-Header e1qr1m9j0"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-pf9rd5-Container{margin:0 auto;max-width:960;padding:1.45rem 1.0875rem;}",
        }
      }
      data-emotion-css="pf9rd5-Container"
    />
    <div
      className="css-pf9rd5-Container e1qr1m9j1"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-1ikcnub-H1{margin:0;}",
          }
        }
        data-emotion-css="1ikcnub-H1"
      />
      <h1
        className="css-1ikcnub-H1 e1qr1m9j2"
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-rkbe0q-StyledLink{color:white;-webkit-text-decoration:none;text-decoration:none;}",
            }
          }
          data-emotion-css="rkbe0q-StyledLink"
        />
        <a
          className="css-rkbe0q-StyledLink e1qr1m9j3"
          href="/"
        >
          Default Starter
        </a>
      </h1>
    </div>
  </header>,
]
`;
